modules/custom/falcon_mail/falcon_mail.info.yml

```diff
@@ -0,0 +1,5 @@
+name: Falcon Mail
+type: module
+description: Provides plugin for mail system with formatter that can do replace tokens.
```

Comment:
...that can wrap emails into HTML templates and replace tokens.

Reply:
done.